ggml.c

```diff
@@ -8567,7 +8764,19 @@ static void ggml_compute_forward(struct ggml_compute_params * params, struct ggm
             {
                 ggml_compute_forward_dup(params, tensor->src0, tensor);
             } break;
-        case GGML_OP_ADD:
+        case GGML_OP_MAP_UNARY:
+            {
+                void (*const fun)(int, float *, float *) = *((void **)tensor->opt[0]->data);
```

Comment:
Maybe name this type something like `ggml_unary_op_t`? (and then name the parameters in that type so that it’s clear what they mean)?

Reply:
Absolutely. At this point, I'm just trying to see if the maintainers would be open to the general idea of adding this type of operation.

The actual implementation can and should be cleaned up before it becomes a real pull. (This is POC is also missing the handling in backward building thing and setting the number of tasks.)